package/yast2-firewall.changes

```diff
@@ -1,8 +1,15 @@
+-------------------------------------------------------------------
+Thu Feb  4 09:14:13 UTC 2021 - Josef Reidinger <jreidinger@suse.com>
+
+- Add to firewall/security proposal option to setup selinux if
+  given product require it. (jsc#SLE-17427)
```

Comment:
NP: I'm using jsc#SMO-20, and jsc#SLE-17342. The latest is the DEV task.

Reply:
too many jira entries. I am using epic mentioned in dev task